descriptors/ModuleDescriptor-template.json

```diff
@@ -998,6 +998,19 @@
         }
       ]
     },
```

Comment:
we should add template-engine as required interface

Reply:
and users interface as well

Reply:
thanks, done